.github/workflows/ci_release.yml

```diff
@@ -0,0 +1,54 @@
+name: CI and Release
```

Comment:
I don't think there's value in running scheduled nightly patch releases of this repo, most likely there will be a tag for every commit since I don't expect changes too often and we don't even use the tags for anything on this repo.